test/test_pubsub.py

```diff
@@ -99,3 +106,41 @@ async def test_unsubscribe_sub_id_not_exists(mock_pubsub_subscriptions):
     assert len(mock_pubsub_subscriptions._subscriptions) == 1
     assert 1 in mock_pubsub_subscriptions._subscriptions
     assert result is False
+
+
+@pytest.fixture()
+def mock_pubsub_publish(mocker):
+    pubsub = PubSub()
+    redis_mock = fakeredis.aioredis.FakeRedis()
+    mocker.patch.object(pubsub, '_redis', redis_mock)
```

Comment:
These lines basically mock redis connection and are the same as ones in `mock_pubsub_subscriptions()`. Maybe we could extract them to a separate function and use in both cases?

Reply:
done